flashinfer/gemm.py

```diff
@@ -968,6 +968,20 @@ def _check_cudnn_fp4_availability():
         )
 
 
+def _is_cublas_fp4_available_in_cudnn():
+    """Check if cuBLAS backend for FP4 GEMM is available in cuDNN."""
+    _check_cudnn_availability()
+
+    # Check cuDNN backend version for FP4 support (requires cudnn_version == 9.11.1 or cudnn_version >= 9.13)
```

Comment:
Can we add a function such as `cudnn_version_greater_equal` and return True when it's greater than 9.11.1?

Reply:
I may miss some context here, but why doesn't `cudnn_version_greater_equal(91101)` works?